docs/content/config/setup.sh.md

```diff
@@ -13,13 +13,9 @@ wget https://raw.githubusercontent.com/docker-mailserver/docker-mailserver/maste
 chmod a+x ./setup.sh
 ```
 
-!!! info
-
-    Make sure to get the `setup.sh` that comes with the release you're using. Look up the release and the git commit on which this release is based upon by selecting the appropriate tag on GitHub. This can done with the "Switch branches/tags" button on GitHub, choosing the right tag. This is done in order to rule out possible inconsistencies between versions.
-
 ## Usage
 
-Run `./setup.sh help` and you'll get some usage information:
+Run `./setup.sh help` and you'll get ~~all you have ever wanted~~ some usage information:
```

Comment:
This should work according to [upstream docs](https://squidfunk.github.io/mkdocs-material/reference/formatting/#highlighting-text).

It uses [this plugin](https://facelessuser.github.io/pymdown-extensions/extensions/tilde/ AFAIK, ah scrolling up on the original upstream docs link [mentions we have to enable it](https://squidfunk.github.io/mkdocs-material/reference/formatting/#caret-mark-tilde) in `mkdocs.yml` config:

```yaml
markdown_extensions:
  - pymdownx.caret
  - pymdownx.mark
  - pymdownx.tilde
```

Add `- pymdownx.tilde` into this part of the config:

https://github.com/docker-mailserver/docker-mailserver/blob/0da66ccb34dcaf0663b6b5635ba97f990b761881/docs/mkdocs.yml#L68

Reply:
Done. [20502294be003363e7f324a8e49166c7085324e4]